### ISSUE TITLE
remotes::install_github("NIVANorge/aquamonitR") not pulling latest version

### ISSUE THREAD

**Issue by JamesSample** [NIVANorge/aquamonitR#7]
Roar recently made some changes to the backend that resulted in some minor changes to column names. I fixed this in 1650a42551d94ffdca7e8487ead05d9a5fa6ac2c, but running

    remotes::install_github("NIVANorge/aquamonitR")

Gives the message

```
Skipping install of 'aquamonitR' from a github remote, the SHA1 (1650a425) has not changed since last install.
  Use `force = TRUE` to force installation
```

Using `force = TRUE` does download and reinstall, but seems to install the old version (i.e. pre 1650a42551d94ffdca7e8487ead05d9a5fa6ac2c), which seems odd. 

Any ideas @RaoulWolf ? Maybe something to do with the way R libraries are packaged?

Thanks!

**Comment by RaoulWolf** [comment]
Hi James, two things come to mind:

1) The changes need to made in [R/get_projects.R](https://github.com/NIVANorge/aquamonitR/blob/main/R/get_projects.R) and not in [aquamonitR.R](https://github.com/NIVANorge/aquamonitR/blob/main/aquamonitR.R). You can actually delete the `aquamonitR.R` file, as all its functionalities have been split-up into separate R functions in the `R` folder; it is ignored during the installation of the package anyway.

2) While the above comment likely will solve the issue (🤞), you may also want to consider a tiny version bump in the [DESCRIPTION](https://github.com/NIVANorge/aquamonitR/blob/main/DESCRIPTION), *i.e.*, `0.0.0.9000` -> `0.0.0.9001`. It could be that the {remotes} package also checks the package version. (Yes I'm too lazy to look it up myself 😄)

Hope this helps!!

EDiT: opened a PR #8

**Comment by JamesSample** [comment]
Closed by #8. Thank Raoul!
